README.md

```diff
@@ -1 +1,14 @@
-# simplefs
\ No newline at end of file
+# simplefs
+
+## Pre-requisites
+
+In order to build this crate install
+[`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/) and fuse
+v2.9+ 
+
+
+on debian
+```bash
+sudo apt-get install libfuse-dev pkg-config
+```
+
```

Comment:
i'll update when i try this on my mac

probably `brew install pkg-config`  and fuse?